can_validator/templates/MY17_Can_Library_BASE.txt

```diff
@@ -0,0 +1,124 @@
+#include "MY17_Can_Library.h"
+#include "evil_macros.h"
+
```

Comment:
why is `MY17_Can_Library_BASE` a `.txt` and not a `.c`?

Reply:
It'd be useful for syntax highlighting, but it could also be included by mistake by some careless developer. `.c` seems better though.

Reply:
Yeah, I had issues with it getting included so I changed it to a `.txt`. Now that I think about it though, I've moved the file structure around and I think it should be fine as a `.c`. I will change it.